test/mock_client_test.exs

```diff
@@ -0,0 +1,4 @@
+defmodule Closex.MockClientTest do
+  use ExUnit.Case, async: true
+  doctest Closex.MockClient
```

Comment:
Bit moot since there are no doctests.

Would be good to have real tests in here. You don't need to add them now, but perhaps a TODO?

Reply:
There are one or two, mostly testing the error functions when handing in `not_found_id`.